docs/docs/en/guide/project/workflow-relation.md

```diff
@@ -0,0 +1,3 @@
+Workflow Relation screen shows all the existing workflows in a project and their status.
+
+![](/img/new_ui/dev/project/work-relation.png)
```

Comment:
Should use related path, and your img is wrong, you should use dolphinscheduler instead of whale